### PR TITLE
Fixed the Storage API for HTML and other exports

### PR DESCRIPTION
**Pull request by SIsilicon**
Also created a start for documenting all the classes.

### CODE REVIEW

File: addons/godot-firebase/database/reference.gd [hunk 1]
```diff
@@ -120,9 +120,9 @@ func get_data() -> Dictionary:
 
 func _get_remaining_path(is_push : bool = true) -> String:
     if !_filter_query or is_push:
-        return _json_list_tag + _query_tag + _auth_tag + Firebase.Auth.auth.idtoken
+        return _json_list_tag + _query_tag + _auth_tag + _auth_obj.auth.idtoken
```
Comment (timjrobinson) [inline addons/godot-firebase/database/reference.gd:123]: Trying out this new code and when using Realtime DB, this `_auth_obj` is null. Same with `_database_obj`. Does it need to be initialized in a different way?

Comment (WolfgangSenff) [reply addons/godot-firebase/database/reference.gd:123]: Sorry, didn't see this - I think this should be working now, right?

Comment (WolfgangSenff) [reply addons/godot-firebase/database/reference.gd:123]: > Trying out this new code and when using Realtime DB, this `_auth_obj` is null. Same with `_database_obj`. Which causes an immediate crash. Does it need to be initialized in a different way?

Sorry, didn't see this - I think this should be working now, right?

Comment (timjrobinson) [reply addons/godot-firebase/database/reference.gd:123]: Yep it was fixed in #129

**Pull request merged**